fs/log/log.go

```diff
@@ -117,17 +124,35 @@ func InitLogging() {
 
 	// Log file output
 	if Opt.File != "" {
-		f, err := os.OpenFile(Opt.File, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0640)
-		if err != nil {
-			log.Fatalf("Failed to open log file: %v", err)
-		}
-		_, err = f.Seek(0, io.SeekEnd)
-		if err != nil {
-			fs.Errorf(nil, "Failed to seek log file to end: %v", err)
+		var logWriter io.Writer
+		if Opt.Rotate {
+			logWriter = &lumberjack.Logger{
+				Filename:   Opt.File,
+				MaxSize:    int(Opt.RotateMaxSize) / 1024 / 1024, // megabytes
+				MaxAge:     Opt.RotateMaxAge,
+				MaxBackups: Opt.RotateMaxBackups,
+				LocalTime:  false,
```

Comment:
Should this be a config parameter too? I know sysadmins love logs in UTC but normal users may prefer them in localtime?

I guess if we have a parameter then users are going to expect it to alter the logs without logrotation also.

Reply:
Yeah, I didn't want to add additional complication here. Perhaps we can leave it as is for now and come back to it if anyone wants it?